server/routes/overview/educationAndTrainingView.ts

```diff
@@ -1,21 +1,17 @@
-import type { PrisonerSummary } from 'viewModels'
+import type { FunctionalSkills, PrisonerSummary } from 'viewModels'
 
 export default class EducationAndTrainingView {
-  constructor(
-    private readonly prisonerSummary: PrisonerSummary,
-    private readonly tab: string,
-    private readonly prisonNumber: string,
-  ) {}
+  constructor(private readonly prisonerSummary: PrisonerSummary, private readonly functionalSkills: FunctionalSkills) {}
 
   get renderArgs(): {
     prisonerSummary: PrisonerSummary
     tab: string
-    prisonNumber: string
+    functionalSkills: FunctionalSkills
```

Comment:
In these view classes, the prison number is not needed (except the main `OverviewView` class); and the tab name can be hardcoded because thats essentially what the whole class is for (ie. you wouldnt use the `EducationAndTrainingView` class when displaying the Support Needs tab)